pkg/events/server/nats.go

```diff
@@ -21,32 +21,41 @@ package server
 import (
 	"fmt"
 
-	"github.com/asim/go-micro/plugins/events/nats/v4"
 	"github.com/cenkalti/backoff"
 	"go-micro.dev/v4/events"
 
-	stanServer "github.com/nats-io/nats-streaming-server/server"
+	"github.com/asim/go-micro/plugins/events/natsjs/v4"
+	nserver "github.com/nats-io/nats-server/v2/server"
 )
 
-// RunNatsServer runs the nats streaming server
+// RunNatsServer starts the nats server and blocks
 func RunNatsServer(opts ...Option) error {
-	natsOpts := stanServer.DefaultNatsServerOptions
-	stanOpts := stanServer.GetDefaultOptions()
+	options := &nserver.Options{}
 
 	for _, o := range opts {
-		o(&natsOpts, stanOpts)
+		o(options)
 	}
-	_, err := stanServer.RunServerWithOpts(stanOpts, &natsOpts)
-	return err
+
+	server, err := nserver.NewServer(options)
+	if err != nil {
+		return err
+	}
+
+	c := &nserver.JetStreamConfig{}
+
+	server.EnableJetStream(c)
```

Comment:
Linter is angry that we don't check the error
```suggestion
	_ = server.EnableJetStream(c)
```